Dockerfile

```diff
@@ -70,12 +38,13 @@ RUN git clone --depth 1 -b 2.4.1 https://github.com/modelon-community/fmi-librar
   && mkdir fmi_build && cd fmi_build \
   && mkdir fmi_library \
   && cmake -DCMAKE_INSTALL_PREFIX=/build/fmi-libary/fmi_library .. \
-  && make -j4 \
+  && make -j$(nproc) \
```

Comment:
This just makes the compilation run with a number of threads equal to what the host system provides to the container.